app/assets/stylesheets/mobile/mobile.scss

```diff
@@ -168,8 +152,7 @@ h3 {
   margin-bottom: 10px;
 
   border: 1px solid #bbb;
-  border-top: 1px solid $border-grey;
-  border-bottom: 1px solid #aaa;
+  border-bottom-color: #aaa;
```

Comment:
Nope. Not right now. This CSS is a mess. It's better to refactor it later than start to spread the mess right now. Actually, I don't remember why I touched this. Probably because RubyMine said me there was something useless. I prefer to keep the changes as minimal as possible here.